devtools/test_dashboard_v2/mocks.js

```diff
@@ -0,0 +1,135 @@
+    { name: 'gl2d - line style', file: 'gl2d_line_style.json' },
+    { name: 'gl2d - multiple subplots', file: 'gl2d_multiple_subplots.json' },
+    { name: 'gl2d - scatter colorscale colorbar', file: 'gl2d_scatter-colorscale-colorbar.json' },
+    { name: 'gl2d - scatter marker line colorscales', file: 'gl2d_scatter-marker-line-colorscales.json' },
+    { name: 'gl2d - show legend', file: 'gl2d_show_legend.json' },
+    { name: 'gl2d - simple inset', file: 'gl2d_simple_inset.json' },
+    { name: 'gl2d - size margins', file: 'gl2d_size_margins.json' },
+    { name: 'gl2d - stacked coupled_subplots', file: 'gl2d_stacked_coupled_subplots.json' },
+    { name: 'gl2d - stacked subplots', file: 'gl2d_stacked_subplots.json' },
+
+    // gl3d
+    { name: 'gl3d - bunny-hull', file: 'gl3d_bunny-hull.json' },
+    { name: 'gl3d - ibm-plot', file: 'gl3d_ibm-plot.json' },
+    { name: 'gl3d - marker-color', file: 'gl3d_marker-color.json' },
+    { name: 'gl3d - log-axis-big', file: 'gl3d_log-axis-big.json' },
+    { name: 'gl3d - delaunay', file: 'gl3d_delaunay.json' },
+    { name: 'gl3d - log-axis', file: 'gl3d_log-axis.json' },
+    { name: 'gl3d - multi-scene', file: 'gl3d_multi-scene.json' },
+    { name: 'gl3d - surface-lighting', file: 'gl3d_surface-lighting.json' },
+    { name: 'gl3d - z-range', file: 'gl3d_z-range.json' },
+    { name: 'gl3d - mirror-ticks', file: 'gl3d_mirror-ticks.json' },
+    { name: 'gl3d - autorange-zero', file: 'gl3d_autorange-zero.json' },
+    { name: 'gl3d - contour-lines', file: 'gl3d_contour-lines.json' },
+    { name: 'gl3d - xy-defined-ticks', file: 'gl3d_xy-defined-ticks.json' },
+    { name: 'gl3d - opacity-surface', file: 'gl3d_opacity-surface.json' },
+    { name: 'gl3d - projection-traces', file: 'gl3d_projection-traces.json' },
+    { name: 'gl3d - opacity-scaling-spikes', file: 'gl3d_opacity-scaling-spikes.json' },
+    { name: 'gl3d - text-weirdness', file: 'gl3d_text-weirdness.json' },
+    { name: 'gl3d - wire-surface', file: 'gl3d_wire-surface.json' },
+    { name: 'gl3d - triangle', file: 'gl3d_triangle.json' },
+    { name: 'gl3d - snowden', file: 'gl3d_snowden.json' },
+    { name: 'gl3d - bunny', file: 'gl3d_bunny.json' },
+    { name: 'gl3d - ribbons', file: 'gl3d_ribbons.json' },
+    { name: 'gl3d - scatter-date', file: 'gl3d_scatter-date.json' },
+    { name: 'gl3d - cufflinks', file: 'gl3d_cufflinks.json' },
+    { name: 'gl3d - chrisp-nan-1', file: 'gl3d_chrisp-nan-1.json' },
+    { name: 'gl3d - marker-arrays', file: 'gl3d_marker-arrays.json' },
+    { name: 'gl3d - scatter3d-colorscale', file: 'gl3d_scatter3d-colorscale.json' },
+    { name: 'gl3d - autocolorscale', file: 'gl3d_autocolorscale.json' },
+    { name: 'gl3d - nan-holes', file: 'gl3d_nan-holes.json' },
+    { name: 'gl3d - scatter3d-connectgaps', file: 'gl3d_scatter3d-connectgaps.json' },
+    { name: 'gl3d - tet', file: 'gl3d_tet.json' },
+    { name: 'gl3d - surface_intensity', file: 'gl3d_surface_intensity.json' }
```

Comment:
Yep. If we want to maintain "clean" names and tags, we could add them as properties of the mock objects too.